src/abstractdataframe/show.jl

```diff
@@ -604,6 +604,11 @@ function _show(io::IO,
 
     compact_printing::Bool = get(io, :compact, true)
 
+    # For consistency, if `kwargs` has `compact_printng`, we must use it.
+    if haskey(kwargs, :compact_printing)
+        compact_printing = kwargs[:compact_printing]
+    end
+
```

Comment:
maybe this will be easier to read? (it is a soft suggestion)
```suggestion
    # For consistency, if `kwargs` has `compact_printng`, we must use it.
    if haskey(kwargs, :compact_printing)
        compact_printing = kwargs[:compact_printing]
    else
        compact_printing::Bool = get(io, :compact, true)
    end

```